components/dashboard/src/settings/selectClass.tsx

```diff
@@ -48,7 +48,7 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
             {supportedClasses.map((c) => {
                 return (
                     <WorkspaceClass
-                        additionalStyles="w-80 h-32"
+                        additionalStyles="w-80"
```

Comment:
Unfortunately, removing the fixed height causses the bottom padding to look enormous 😞 but I didn't have a good suggestion for how to get it right without significant changes. So, maybe let's iterate on the design of the workspace class selector in follow-ups.

Reply:
by enormous you mean more than what is shown in screenshot in the initial comment?

Reply:
I mean the empty space between the dot and the bottom border of the card:

## Before

<img width="688" alt="Screenshot 2022-12-12 at 15 09 52" src="https://user-images.githubusercontent.com/599268/207067053-38e6d1e4-bdfe-4e67-80cc-4b71343d4c27.png">

## After

<img width="684" alt="Screenshot 2022-12-12 at 15 10 10" src="https://user-images.githubusercontent.com/599268/207067061-41dcf303-90b8-46ff-97a1-fa89624677a0.png">

Reply:
Ah, ok that is intentional. cc @gtsiolis

Reply:
Padding changes look ok. ✔️ 

I'd separate the consumption rate (10 credits/hour) from the resources (dot) indicator (● ●) information to scale better, but can be done this later when we introduce more than two workspace classes.

Thanks for the ping, @svenefftinge! 🏓